IOPool/Input/src/ProvenanceAdaptor.cc

```diff
@@ -19,11 +19,9 @@ ProvenanceAdaptor.cc
 #include "DataFormats/Provenance/interface/ProductRegistry.h"
 #include "FWCore/Utilities/interface/Algorithms.h"
 namespace edm {
-  
-  void
-  ProvenanceAdaptor::fixProcessHistory(ProcessHistoryMap& pHistMap,
-				       ProcessHistoryVector& pHistVector) {
-    assert (pHistMap.empty() != pHistVector.empty());
+
+  void ProvenanceAdaptor::fixProcessHistory(ProcessHistoryMap& pHistMap, ProcessHistoryVector& pHistVector) {
```

Comment:
Some places it is aligning the pointer and reference symbols left. Maybe I just do not understand the algorithm it is using to decide ...